### PR TITLE
Adds additional unit test for service bind

### PR DESCRIPTION
**Pull request by anweshan**

### CODE REVIEW

File: tests/actions/assistant-v1/message.test.js [hunk 1]
```diff
@@ -86,8 +86,18 @@ describe('message', () => {
...
       })
-      .catch(() => {
-        assert.fail('Failure on valid payload');
+      .catch((e) => {
+        assert.fail(e);
       });
   });
+  it('should fail with __bx_creds as credential source with a different name', () => {
+    const params = { test: 'hello', __bx_creds: { assistant: payload } };
+    console.log(params);
```
Comment (dpopp07) [inline tests/actions/assistant-v1/message.test.js]: Did you mean to commit this logging statement?

```suggestion
```

**Comment by anweshan** [comment]
@dpopp07 removed the console.log statement in tests

**Comment by anweshan** [comment]
@dpopp07 is there a reason why this hasn't been reviewed yet?

**Comment by dpopp07** [comment]
It's just been stuck in my backlog, I will get to it this week for sure.

**Review by dpopp07 — APPROVED** [review]
Sorry it took me a while to get to this - looks good 👍

**Pull request merged**